packs/spell-effects/spell-effect-draw-ire.json

```
@@ -0,0 +1,107 @@
+{
+    "_id": "0WYy3bY7HJ7TDSYD",
+    "img": "icons/skills/targeting/target-glowing-yellow.webp",
+    "name": "Spell Effect: Draw Ire",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Draw Ire]</p>\n<p><strong>Failure</strong> The target takes full damage and the penalty.</p>\n<p><strong>Critical Failure</strong> The target takes double damage, and the status penalty is -2.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Draw Ire]</p>\n<p>The target takes a -1 status penalty to attack rolls. On a critical failure the penalty is -2.</p>"
```
Effects should only say what they actually automate, leaving the rest to the source item.